internal/connector/connector.go

```diff
@@ -361,7 +362,7 @@ func (cw *wrapper) Close() error {
 
 	if cw.serverStdin != nil {
 		err = cw.serverStdin.Close()
-		if err != nil {
+		if err != nil && !strings.Contains(err.Error(), "already closed") {
```

Comment:
`errors.Is(err, os.ErrClosed)`?

Reply:
Ah yes, that's a better check. I'll push a fix.